### ISSUE TITLE
Improve Blog CSS

### ISSUE THREAD

**Issue by kylepolich** [data-skeptic/dataskeptic.com#626]
You made some nice suggestions based on github previously.  Let's move forward with that.  

Specifically, I want tables to look good by default.  See this post:

https://dataskeptic.com/blog/tools-and-techniques/2018/using-cross-join-lateral-for-improved-json-handling-in-postgres

Tables look terrible.  Let's add some style by default for all Blog pages.

**Comment by gliblozhkovyi** [comment]
One thing could be unclear after this PR. I have adjusted regular post text size. Now it a little bit bigger.  It looks like a Medium post right now. Is that OK?

However, reference post typography now looks pretty awesome. 

We could adjust things with images gallery, custom code format components, e.t.c. as mentioned before

**Comment by kylepolich** [comment]
Yes, the text looks good!  I personally am not very good at noticing good design, typefacing, etc.  So I appreciate these suggestions very much.

Could you provide a sample of image gallery and custom code components?
